### PR TITLE
Rename Buy and Sell to Long and Short

### PR DESCRIPTION
**Pull request by DeliciousHair**
Addresses #373. Change made for consistency of language and removes some ambiguity in the business logic as a side-effect.

### CODE REVIEW

File: daemon/src/model/cfd.rs [hunk 1]
```diff
@@ -1301,28 +1301,33 @@ impl AsBlocks for Duration {
...
+fn calculate_short_margin(price: Price, quantity: Usd) -> Amount {
     quantity / price
 }
 
-fn calculate_liquidation_price(leverage: Leverage, price: Price) -> Price {
+fn calculate_long_liquidation_price(leverage: Leverage, price: Price) -> Price {
     price * leverage / (leverage + 1)
 }
 
+// PLACEHOLDER
```
Comment (klochowicz) [inline daemon/src/model/cfd.rs:1326]: nit: prefer `TODO`

File: frontend/src/TakerApp.tsx [hunk 2]
```diff
@@ -206,7 +206,7 @@ export default function App() {
                                     makeNewOrderRequest(payload);
                                 }}
                             >
-                                BUY
+                                LONG
```
Comment (klochowicz) [inline frontend/src/TakerApp.tsx]: IMHO to be meaningful to the user, this button should name an action its doing, ie. use a verb. "Go long", "Take", or even "Buy" (as it used to be)? Or anything else that would be less confusing than pure "Long".

**Review by klochowicz — APPROVED** [review]

File: frontend/src/TakerApp.tsx (hunk 2 above)
Comment (DeliciousHair) [reply frontend/src/TakerApp.tsx]: Is this the text that appears on the button?

File: daemon/src/model/cfd.rs (hunk 1 above)
Comment (DeliciousHair) [reply daemon/src/model/cfd.rs:1326]: It's not really a TODO though, but literally is a placeholder that has meaning + functionality once maker-goes-long is implemented.

File: frontend/src/TakerApp.tsx (hunk 2 above)
Comment (klochowicz) [reply frontend/src/TakerApp.tsx]: indeed :) this is the string that's visible in the UI. the simplest thing would be to keep it as "Buy", but I'm happy with anything that makes sense, we can adjust it later as well.

Comment (DeliciousHair) [reply frontend/src/TakerApp.tsx]: NP, I didn't realize that was the visual text!

**Review by da-kami — APPROVED** [review]

**Pull request merged**